test/Microsoft.AspNetCore.SignalR.Client.FunctionalTests/HubConnectionTests.cs

```diff
@@ -41,6 +41,12 @@ public HubConnectionTests(ServerFixture<Startup> serverFixture, ITestOutputHelpe
             _serverFixture = serverFixture;
         }
 
+        public IDisposable StartLog2(out ILoggerFactory loggerFactory, LogLevel logLevel = LogLevel.Trace, string testName = null)
```

Comment:
`StartLog2` WAT? Let's get a better name there :)

Reply:
I'm not checking this in